setup.py

```diff
@@ -4,30 +4,27 @@
 from setuptools import find_packages, setup
 
 GARAGE_GH_TOKEN = os.environ.get('GARAGE_GH_TOKEN') or 'git'
-GYM_VERSION = '0.15.4'
+GYM_VERSION = '0.17.2'
 
 # Required dependencies
 REQUIRED = [
     # Please keep alphabetized
     'akro',
     'click>=2.0',
-    'cloudpickle<1.5',
+    'cloudpickle==1.3',
     'cma==2.7.0',
     'dowel==0.0.3',
     f'gym[atari,box2d,classic_control]=={GYM_VERSION}',
     'numpy>=1.14.5',
     'psutil',
-    # Pyglet 1.4.0 introduces some api change which breaks some
-    # gym environments
-    # See: https://github.com/openai/gym/issues/1588
-    'pyglet<1.4.0,>=1.3.0',
+    'pyglet>=1.4.0,<=1.5.0',
```

Comment:
I think we can remove this as a direct dependency if we are upgrading gym

Reply:
I copy this form gym's [setup.py](https://github.com/openai/gym/blob/6df1b994bae791667a556e193d2a215b8a1e397a/setup.py#L31)